tfhe/src/typed_api/config.rs

```diff
@@ -113,6 +113,15 @@ impl ConfigBuilder {
         self
     }
 
+    #[cfg(feature = "integer")]
+    pub fn enable_default_uint8_small(mut self) -> Self {
+        let mut params = crate::typed_api::integers::FheUint8Parameters::default();
```

Comment:
this will need to use the SMALL variant of parameters they are available in crate::shortint::parameters

like PARAMS_SMALL_MESSAGE_2_CARRY_2

Reply:
same for other uint types

Reply:
Didn't see they existes

Reply:
no worries, they are rather discreet